src/Components/ProvisioningWizard/steps/FinishProgress/index.js

```diff
@@ -167,15 +165,15 @@ const FinishStep = ({ onClose, imageID }) => {
               </span>
             </span>
           </EmptyStateBody>
+          {isError && (
+            <Button onClick={() => goToStepById(1)} variant="primary">
+              Edit
+            </Button>
+          )}
           <EmptyStateSecondaryActions>
-            {isError && (
-              <Button onClick={() => goToStepById(1)} variant="link">
-                Back
-              </Button>
-            )}
             <Button
               variant="link"
-              isDisabled={!showCloseButton()}
+              isDisabled={!isError && activeStep < 2}
```

Comment:
what if there's an error during step 2 and above? shouldn't we have a clickable close button?